cmd/kured/main.go

```diff
@@ -236,13 +251,16 @@ func release(lock *daemonsetlock.DaemonSetLock) {
 
 func drain(client *kubernetes.Clientset, node *v1.Node) {
 	nodename := node.GetName()
-
 	log.Infof("Draining node %s", nodename)
-
 	if slackHookURL != "" {
 		if err := slack.NotifyDrain(slackHookURL, slackUsername, slackChannel, nodename); err != nil {
 			log.Warnf("Error notifying slack: %v", err)
 		}
+	} 
+	if notifyURL != "" {
+		if err := shoutrrr.Send(notifyURL, fmt.Sprintf("Draining node %s", nodename)); err != nil {
+			log.Warnf("Error notifying: %v", err)
+		}
```

Comment:
This will be conflicting with the other PR which allow user-defined text for notifications. We'll need to keep it in mind.